src/Plugin.php

```diff
@@ -39,6 +39,18 @@ class Plugin {
    */
   private static $baseUrl;
 
+  /**
+   * @implements plugins_loaded
+   */
+  public static function plugins_loaded() {
+    // Change URLs pointing to /wp-login.php into /login.php.
+    add_filter('network_site_url', __NAMESPACE__ . '\Security::site_url', 100, 3);
```

Comment:
Wondering if this should an even higher priority than 100, e.g. 999 in case other plugins rewrite it? Even if I'm not aware of hooks rewriting the URLs in our projects we should consider this is a public plugin which may is used by other users as well.

Reply:
100 is relatively unusual already, I think we should rather increase it when we run into issues.